code/style.css

```diff
@@ -5,3 +5,326 @@ this can be used for the FAQ-accordion. read more here: https://www.w3schools.co
+  flex-wrap: wrap;
+  justify-content: center;
+  background-color: #578378;
+  align-items: center;
+  width: 95%;
+  border-radius: 5px;
+  position: static;
+  margin: 30px 0px 30px 0px;
+  box-shadow: 5px 5px 5px #7A8C8E;
+}
+label{
+ margin: 10px 10px 0px 0px;
+ font-size: 15px;
+ font-weight: 300;
+ color: white;
+}
+input{
+  margin: 5px;
+  width: auto;
+  padding: 5px;
+  border-radius: 5px;
+  border: none;
+
+}
+input:checked{
+  box-shadow: 3px 3px 3px #356B71;
+}
+input:focus{
+  background-color: honeydew;
+}
+
+select{
+  margin: 5px;
+  width: 95%;
+  padding: 5px;
+  border-radius: 5px;
+  border: none;
+}
+.text-field{
+  justify-content: space-between;
+  display: flex;
+  flex-wrap: wrap;
+  flex-direction: column;
+}
+
+.button{
+  width: 80%;
+  border-radius: 5px;
+  border: 1px solid white;
+  box-shadow: 2px 2px 3px #49787e;
+  font-size: 17px;
+  font-weight:100;
+  background-color: #3C767D;
+  color: white;
+  justify-content: center;
+  align-items: center;
+  margin: 20px;
+  padding: 7px;
+}
+.header-button{
+  border-radius: 5px;
+  border: 1px solid white;
+  box-shadow: 2px 2px 3px #49787e;
+  font-size: 17px;
+  font-weight:100;
+  background-color: #3C767D;
+  color: white;
+  justify-content: center;
+  align-content: flex-end;
+  margin: 20px;
+  padding: 7px;
+  
+}
+a{
+  text-decoration: none;
+  color: white;
+}
+
+.button-wrapper{
+  align-content: center;
+  display: flex;
+  justify-content: center;
+}
+.button:hover{
+  background-color: #4D7377;
+  box-shadow: 2px 2px 3px white;
+}
+  .accordion{
+  cursor: pointer;
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  justify-content: center;
+  background-color: #577e83;
+  align-items: center;
+  width: 95%;
+  border-radius: 5px;
+  position: static;
+  margin-top: 30px;
+  box-shadow: 5px 5px 5px #7A8C8E;
+  margin: 30px 0px 30px 0px;
+}
+  .question{
+    background-color: #578378;
+    margin: auto;
+    width: 100%;
+    font-size: 17px;
+    color: white;
+    border-radius: 0px 0px 5px 5px;
+    border-top: 1px solid white;
+    display: flex;
+   justify-content: space-between;
+    align-items: center;
+  }
+
+  .answer{
+    background-color:#6F6854;
+    width: 100%;
+    margin: auto;
+    overflow: hidden;
+    margin: 0;
+    /* border-top: 1px solid white; */
+    transition: 0.7s ease-out;
+    max-height: 0;
+
+  }
+
+  .active + .answer {
+    display: block;
+    max-height: 200px;
+
+  }
+  
+  .footer-wrapper{
+  display: flex;
+  text-align: center;
+  flex-direction: column;
+  background-color: #577e83;
+  width: 100%;
+  padding: 30px 0px;
+  align-items: center;
+  justify-content: center;
+  bottom:  0px;
+}
+
+.icon{
+  filter: invert(100%);
+  width: 20px;
+  height: 20px;
+  transform: rotate(270deg);
+  float: right;
+  margin-right: 20px;
+}
+
+.active .icon{
+  transform: rotate(90deg);
+  transition: 0.5s ;
+}
+
+
+/*mobile*/
+@media (min-width:0px) and (max-width:767px){
+ 
```

Comment:
You did it so well of making the code clear so media queries would only have the variables, and not overlapped. I really should look at this and learn! Mine is way messy!